tests/lib/prepare.sh

```diff
@@ -55,7 +55,7 @@ setup_systemd_snapd_overrides() {
 [Unit]
 $START_LIMIT_INTERVAL
 [Service]
-Environment=SNAPD_DEBUG_HTTP=7 SNAPD_DEBUG=1 SNAPPY_TESTING=1 SNAPD_CONFIGURE_HOOK_TIMEOUT=30s SNAPPY_USE_STAGING_STORE=$SNAPPY_USE_STAGING_STORE
+Environment=SNAPD_DEBUG_HTTP=7 SNAPD_DEBUG=1 SNAPPY_TESTING=1 SNAPD_REBOOT_DELAY=+10 SNAPD_CONFIGURE_HOOK_TIMEOUT=30s SNAPPY_USE_STAGING_STORE=$SNAPPY_USE_STAGING_STORE
```

Comment:
Can we make it consistent with how SNAPD_CONFIGURE_HOOK_TIMEOUT is handled, e.g. support "s" suffix etc (you will need to use time.ParseDuration() for anything other than "now")

Reply:
done,  we don't need to special case now, 0s or 0 would work for that